src/ui/zcl_abapgit_html_form.clas.abap

```diff
@@ -434,18 +439,21 @@ CLASS zcl_abapgit_html_form IMPLEMENTATION.
 
       WHEN zif_abapgit_html_form=>c_cmd_type-button.
 
-        ii_html->add( |<button type="submit" name="action" value="{
-          is_cmd-action }" class="action-commands">{ is_cmd-label }</button>| ).
+        ii_html->add( |<button type="submit"|
+                   && | name="action" value="{ is_cmd-action }"|
+                   && | class="action-commands">{ is_cmd-label }|
+                   && |</button>| ).
 
       WHEN zif_abapgit_html_form=>c_cmd_type-input.
 
-        ii_html->add( |<input type="submit" value="{
-          is_cmd-label }" formaction="sapevent:{ is_cmd-action }">| ).
+        ii_html->add( |<input type="submit"|
+                   && | value="{ is_cmd-label }"|
+                   && | formaction="sapevent:{ is_cmd-action }">| ).
 
       WHEN zif_abapgit_html_form=>c_cmd_type-input_main.
 
-        ii_html->add( |<input type="submit" value="{
-          is_cmd-label }" class="main" formaction="sapevent:{ is_cmd-action }">| ).
```

Comment:
I think abapLint is right, "{" should not be split from it's contents in ABAP. Suggest splitting it with && like the other statements: 
```
        ii_html->add( |<input type="submit" value="{ is_cmd-label }" | 
                   && |class="main" |
                   && |formaction="sapevent:{ is_cmd-action }">| ).
```

Reply:
abaplint only suggests moving it to one line if it can fit into the max 

120 characters is default, https://rules.abaplint.org/keep_single_parameter_on_one_line/

Reply:
OK, I can fix it later today so all lights are green.